src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala

```diff
@@ -11,7 +12,10 @@ import org.fortysevendeg.sparksftp.config.model.configs.{ReadingSFTPConfig, SFTP
 import org.training.trainingbot.config.ConfigLoader
 import org.fortysevendeg.sparksftp.config.model.configs
 
+import scala.concurrent.ExecutionContext
+
 object ReadingSFTPConnectorApp extends IOApp {
+  implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
An `IOApp` gives you a `ContextShift` for free

Reply:
Good catch!
Changed in be83297